src/main/java/com/sprint/mission/discodeit/service/basic/BasicChannelService.java

```diff
@@ -1,165 +1,124 @@
 package com.sprint.mission.discodeit.service.basic;
 
+import com.sprint.mission.discodeit.dto.ChannelDto;
+import com.sprint.mission.discodeit.dto.PrivateChannelCreateRequest;
+import com.sprint.mission.discodeit.dto.PublicChannelCreateRequest;
+import com.sprint.mission.discodeit.dto.PublicChannelUpdateRequest;
 import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.entity.ChannelType;
 import com.sprint.mission.discodeit.entity.Message;
-import com.sprint.mission.discodeit.entity.User;
+import com.sprint.mission.discodeit.entity.ReadStatus;
+import com.sprint.mission.discodeit.repository.ChannelRepository;
+import com.sprint.mission.discodeit.repository.MessageRepository;
+import com.sprint.mission.discodeit.repository.ReadStatusRepository;
 import com.sprint.mission.discodeit.service.ChannelService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
 
-
-
-
-import java.io.BufferedReader;
-import java.io.FileReader;
-import java.io.IOException;
-import java.lang.reflect.InvocationTargetException;
-import java.lang.reflect.Method;
+import java.time.Instant;
 import java.util.*;
 
-/**
- * Hence Circular Reference, deprecate use JCFs as memberfield of JCF.
- * Instead of composition, refactor JCFs using Singleton pattern.
- */
+@RequiredArgsConstructor
+@Service
 public class BasicChannelService implements ChannelService {
-    private static BasicChannelService instance;
-    private Map<UUID, Channel> ChannelList;
+    private final ChannelRepository channelRepository;
+    private final ReadStatusRepository readStatusRepository;
+    private final MessageRepository messageRepository;
 
-    public BasicChannelService() {
-        this.ChannelList = new HashMap<>();
-    }
+    @Override
+    public Channel createChannel(PublicChannelCreateRequest request) {
```

Comment:
메서드명에도 Public 채널을 생성한다는 의미를 넣어볼까요?  
```suggestion
    public Channel createPublicChannel(PublicChannelCreateRequest request) {
```